components/parser/src/parser2/lite_parse.rs

```diff
@@ -689,18 +694,20 @@ mod tests {
     use crate::parser::reporting::print_parse_error;
     use crate::parser::{Span, Spanned};
     use crate::parser2::macros::{macros, Macros};
-    use crate::parser2::quicklex::{Token, Tokenizer};
+    use crate::parser2::quicklex::Tokenizer;
     use crate::parser2::test_helpers::{process, Annotations, Position};
+    use crate::LexToken;
 
     use log::trace;
     use std::collections::HashMap;
     use unindent::unindent;
 
+    /*
     #[test]
     fn test_lite_parse() {
         return;
         crate::init_logger();
-
+    
```

Comment:
You might want to run a format pass over this... I *think* it would clean up the stray spaces